opennmt/models/sequence_to_sequence.py

```diff
@@ -451,7 +469,7 @@ def print_prediction(self, prediction, params=None, stream=None):
             raise ValueError(
                 "with_alignments is set but the model did not return alignment information"
             )
-        num_hypotheses = params.get("n_best", len(prediction["log_probs"]))
+        num_hypotheses = len(prediction["log_probs"])
```

Comment:
This reverts a change in master, probably due to an incorrect merge. You should not change this line.

Reply:
Yup, merge mistake. I reverted back to how it was in the latest commit